roles/bwc/tasks/ldap.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- name: Setup st2.conf auth backend to LDAP
+  become: yes
+  ini_file:
+    dest: "{{ st2_config_file_path }}"
+    section: auth
+    option: backend
+    value: ldap
+    backup: yes
```

Comment:
Yeah, we also have to set the backend_kwargs before restarting. See next step. I did notify.